clients/src/main/java/org/apache/kafka/common/record/DefaultRecordBatch.java

```diff
@@ -91,9 +91,9 @@
  *
  * The current attributes are given below:
  *
- *  -------------------------------------------------------------------------------------------------
- *  | Unused (6-15) | Control (5) | Transactional (4) | Timestamp Type (3) | Compression Type (0-2) |
- *  -------------------------------------------------------------------------------------------------
+ *  ---------------------------------------------------------------------------------------------------------------------------
```

Comment:
Could we add some doc on how Delete Horizon flag is being used?

Reply:
Done.